.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: ruby
+rvm:
+  - 2.3.4
```

Comment:
missing actual ruby versions, (2.5.0, 2.4.3) …

and it is good practice to use the latest version of a specific branch (2.3.6 instead of 2.3.4)